packages/sdks-tests/src/tests/helpers/visual-editor.ts

```diff
@@ -10,13 +10,15 @@ export const launchEmbedderAndWaitForSdk = async ({
   basePort,
   path,
   gotoOptions,
+  messageCondition,
 }: {
   page: Page;
   basePort: number;
   path: string;
   gotoOptions?: Parameters<Page['goto']>[1];
+  messageCondition: (msg: ConsoleMessage) => any;
 }) => {
-  const msgPromise = page.waitForEvent('console', msg => msg.text() === SDK_LOADED_MSG);
+  const msgPromise = page.waitForEvent('console', msg => messageCondition(msg));
   await page.goto(getEmbeddedServerURL(path, basePort), gotoOptions);
-  await msgPromise;
```

Comment:
I don't think you needed to change this functional at all, or make it receive a `messageCondition`.
We want this function to always wait for `SDK_LOADED_MSG` before returning.

What I would do is:

- revert the changes to this function
- for the custom component tests that need to check for other messages, I would add that logic there specifically:


```ts
const customComponentMsgPromise = page.waitForEvent('console', msg => msg.text().includes(HELLO_CUSTOM_COMPONENT_LOADED_MESSAGE));
await launchEmbedderAndWaitForSdk(/**/);
await customComponentMsgPromise;
```

Reply:
ahh yes! this is so much simpler. stupid me 😅

have made the changes